moderation/NicknameModeration.go.tmpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+	Custom Command for nickname moderation
+	If a user of your server has an inappropriate or unpingable nickname, mods often have to change their nickname manually.
+	With this custom command, they can moderate nicknames with a simple command.
+	After you moderated the nickname, the nickname will be changed when the moderated user sends their next message.
+	The command nick-numberreset makes the next moderated nickname start at number one.
+
+	Trigger type: Regex
+	Trigger: \A
+
+	Usage:
+	-modnick <@user/ID>
+	-unmodnick <@user/ID>
+	-nick-numberreset
+
+	Credit: 2x2Master#7363
```

Comment:
If you really want, we could keep it like that, but it is recommended to instead use your GitHub Account, as keeping up with potential tag or username changes on Discord is rather tedious:
```suggestion
	Credits: 2x2Master1 <https://github.com/2x2master1>
```